tests/test_domain_std.py

```diff
@@ -252,7 +252,17 @@ def test_glossary_alphanumeric(app):
     restructuredtext.parse(app, text)
     objects = list(app.env.get_domain("std").get_objects())
     assert ("1", "1", "term", "index", "term-1", -1) in objects
-    assert ("/", "/", "term", "index", "term-0", -1) in objects
+    assert ("/", "/", "term", "index", "term-2", -1) in objects
```

Comment:
Hmm... node-id has been changed with this change... I think this is a breaking change. I'll take a look this later.